data/_models/shared/Datasets.csv

```diff
@@ -1,12 +1,19 @@
 tableName,tableExtends,columnName,columnType,key,required,refSchema,refTable,refLink,refBack,validation,semantics,description,profiles,visible
-Datasets,,,,,,,,,,,dcat:Dataset,Definition of a dataset,DataCatalogueFlat,
-Datasets,,collection,ref,1,TRUE,,Collections,,,,dcterms:isPartOf,collection that these variables are part of,DataCatalogueFlat,
-Datasets,,name,,1,TRUE,,,,,,dcterms:title,unique dataset name in the model,DataCatalogueFlat,
+Datasets,,,,,,,,,,,"dcat:Dataset,dct:Dataset",Definition of a dataset,DataCatalogueFlat,
+Datasets,,collection,ref,1,TRUE,,Collections,,,,"dcterms:isPartOf,dct:inSeries",collection of which the dataset is part,DataCatalogueFlat,
```

Comment:
nice

zouden we trouwens CollectionEvent ook als een dct:Dataset moeten zien?

Reply:
Goede suggestie, die doe ik ook